x-pack/plugins/maps/public/classes/sources/es_source/es_source.ts

```diff
@@ -335,15 +447,17 @@ export class AbstractESSource extends AbstractVectorSource {
     return resp.aggregations;
   }
 
-  getValueSuggestions = async (field, query) => {
+  getValueSuggestions = async (field: IField, query: string): Promise<string[]> => {
     try {
       const indexPattern = await this.getIndexPattern();
+      const fieldType = indexPattern.fields.getByName(field.getRootName())!;
```

Comment:
Can you rename to something other then `fieldType`. `fields.getByName` returns IField which includes a type property to calling the results `fieldType` is not very clear